WTWSMS/decisions/minor_decisions.txt

```diff
@@ -240,18 +240,25 @@ decisions = {
 			OR = {
 				religion_group = christian
 				religion_group = pagan_group
+				religion = tzadikite # Martial prowess and hunting was critical
+				culture_group = altaic
+				is_nomadic = yes # Nomads lived for this kind of thing
 			}
 			NOT = {
-				OR = {
```

Comment:
Wait. That is correct. I removed the OR tag because

NOT = {
     OR = {
     }
}

Is the same as 

NOT = { }